aten/src/ATen/native/native_functions.yaml

```diff
@@ -13028,3 +13021,9 @@
   structured_inherits: TensorIteratorBase
   structured: True
   variants: function
+
+# Aux function used in the test TestPythonDispatch.test_kwarg_only_and_positional_default
+# within test/test_python_dispatch.py
+- func: _foobar(Tensor self, bool arg1=True, bool arg2=True, *, bool arg3=True) -> Tensor
```

Comment:
nit: would be modestly better for the function name to say what its intended meaning is